config/main.py

```diff
@@ -2272,20 +2272,20 @@ def breakout(ctx, interface_name, mode, verbose, force_remove_dependencies, load
                       remains unchanged to limit the traffic impact """
 
     click.secho("\nAfter running Logic to limit the impact", fg="cyan", underline=True)
-    matched_item = [intf for intf, speed in del_intf_dict.items() if intf in add_intf_dict.keys() and speed == add_intf_dict[intf]]
+    matched_item = [intf for intf, speed in list(del_intf_dict.items()) if intf in list(add_intf_dict.keys()) and speed == add_intf_dict[intf]]
```

Comment:
```suggestion
    matched_item = [intf for intf in del_intf_dict if intf in add_intf_dict and speed == add_intf_dict[intf]]
```

Reply:
In your suggestion, `speed` is not defined. I believe the correct way to remove the `list()` conversions would be:

```python
matched_item = [intf for intf in del_intf_dict if intf in add_intf_dict and del_intf_dict[intf] == add_intf_dict[intf]]
```

Reply:
I have committed the change I described above.